README.md

```diff
@@ -67,6 +67,12 @@ To provide interoperability with other file formats or applications lacking a
 native integration, the opentimelineio community has built a number of python
 adapter plugins. This includes Final Cut Pro XML, AAF, CMX 3600 EDL, and more.
 
+**Note: v0.16 will be the last release of the
+[OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/)
+to include the full set of adapter plugins. If you rely on these please use the
+[OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/)
+to continue to receive them. Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**
+
```

Comment:
I think rearranging this a bit more in the positive might help:

```suggestion
**Note: for releases after v0.16, users of the [OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/) will also need to install a separate [OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/) to include the full set of adapter plugins.  Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**

For more information: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1386
```

Reply:
I like the change overall, I just worry that the wording might imply you need to install both (which you don't).

What if we do this:

```
**Note: for releases after v0.16, the [OpenTimelineIO PyPI package](https://pypi.org/project/OpenTimelineIO/) will only include the core libraries and file formats. Users that need the full set of adapter plugins should use the [OpenTimelineIO-Plugins PyPI Package](https://pypi.org/project/OpenTimelineIO-Plugins/). Each OpenTimelineIO release will have a matching OpenTimelineIO-Plugins release.**

For more information: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1386
```